crates/bevy_pbr/src/decal/forward_decal.wgsl

```diff
@@ -46,7 +46,7 @@ fn get_forward_decal_info(in: VertexOutput) -> ForwardDecalInformation {
     let uv = in.uv + delta_uv;
 
     let world_position = vec4(in.world_position.xyz + V * diff_depth_abs, in.world_position.w);
-    let alpha = saturate(1.0 - normal_depth * depth_fade_factor);
+    let alpha = saturate(1.0 - (normal_depth / max(depth_fade_factor, 0.001)));
```

Comment:
Would be nice to move this out of the shader and into the host code, and just pass it in as already `1 / max(factor, 0.001)`. Not a huge deal though I guess.

Or maybe we can remove the `1.0 -` or switch the order of things to avoid the division? Too late in the day for me to do algebra 😅 . But I feel like since it's surrounded in saturate() we should be able to do something here.